docs/_docs/monitoring-metrics/metrics.adoc

```diff
@@ -308,6 +308,22 @@ group=<Cache Name>,name="org.apache.ignite.internal.processors.cache.CacheLocalM
 |===
 --
 
+=== Monitoring Processed Keys for Rebuilding Indexes
+
+To get an estimate on how long it takes to rebuild cache indexes, you can use one of the metrics listed below:
+
+. `IsIndexRebuildInProgress` - gives the number of currently rebuilding cache indexes
+. `IndexBuildCountPartitionsLeft` - gives the remaining number of partitions (by cache group) to rebuild indexes for
+
+These metrics allow to estimate an approximate number of indexes left to rebuild.
```

Comment:
It seems that it can only be said about the **IndexBuildCountPartitionsLeft** that it gives little information. And about the **IsIndexRebuildInProgress** just say below.